nixos/modules/config/console.nix

```diff
@@ -34,14 +34,17 @@ let
       "/share/unimaps"
     ];
   };
-
-  setVconsole = !config.boot.isContainer;
 in
 
 {
   ###### interface
 
   options.console  = {
+    enable = mkEnableOption (lib.mdDoc "vconsole") // {
```

Comment:
Some description of what this module does exactly wouldn't hurt.
Like "Enable configuration of the Linux virtual console".

Reply:
mkEnableOption will generate the following right now ``Whether to enable vconsole.
``

Reply:
I know, and it's not very hepful.

Reply:
No, this will disable that systemd will setup virtual consoles so they are no longer available.

Reply:
I know what it does, I wrote the module.
I'm telling you the description "enable vconsole" is next to useless.

Reply:
> Like "Enable configuration of the Linux virtual console".

pushed that